packages/core/src/Component.ts

```diff
@@ -0,0 +1,8 @@
+// Copyright IBM Corp. 2017. All Rights Reserved.
+// Node module: @loopback/core
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+export interface Component {
```

Comment:
Follow up issue for refactor at https://github.com/strongloop/loopback-next/issues/284